addons/project/security/project_security.xml

```diff
@@ -108,7 +108,7 @@
         <field name="groups" eval="[(4, ref('base.group_portal'))]"/>
     </record>
 
-    <record id="project_task_rule_portal" model="ir.rule">
+    <record id="r" model="ir.rule">
```

Comment:
Nope.

Reply:
why not? :'( look's like a good idea!!! Fixed ;)